README.md

```diff
@@ -12,13 +12,21 @@ The contents of this code of conduct apply to our interactions in various areas
 
 ## Values
 
-We want our team to be fun, productive, and safe for all members. In addition to the Social Tables [core values](https://www.socialtables.com/company/about/#values), there are several ways in which our Engineering team distinguishes itself.
+We want our team to be fun, productive, and safe for all members. We've adopted the Social Tables [core values](https://www.socialtables.com/company/about/#values), in ways that specifically apply to the craft of Software Engineering at Social Tables:
 
-**We work with and alongside other teams at Social Tables.** Customer Success, Sales, Marketing, and Operations team members and others across the company are our peers and collaborators, not our clients. We maintain open lines of communication and foster good working relationships—indeed, friendships—with our colleagues. That collaboration is core to our success.
+**We recognize that "It Takes a Village" as we work alongside other teams.** Customer Success, Sales, Marketing, and Operations team members and others across the company are our peers and collaborators, not our clients. We maintain open lines of communication and foster good working relationships—indeed, friendships—with our colleagues. That collaboration is core to our success.
 
-**We value so-called “soft” skills as much as “hard” skills.** That is, we love code, but we recognize that writing code is not the only or most important way to contribute to great products. In fact, we object to the labels “soft” and “hard” because they imply one set of skills is more valuable than another. “Non-technical” is not synonymous with “inferior.”
+**"Hospitality Matters"**  We value “soft” skills as much as “hard” and while we love code, we recognize that writing code is not the only or most important way to contribute to great products. “Non-technical” is not synonymous with “inferior.”
 
-**We value diversity.** Diversity not only makes us a happier team, but evidence shows that diversity makes us a better, more productive team, too.
+**"Data is our Diplomat"**  We defer to data-driven design and decision making. We don't always have the luxury of definitive data and often need to make tough decisions in its absence, but where appropriate data exists we let it drive.
+
+**"Always Honest"**  We crave a work environment conducive to open and honest feedback. We strive for [Radical Candor](http://firstround.com/review/radical-candor-the-surprising-secret-to-being-a-good-boss/) and recognize that both Obnoxious Aggression and Ruinous Empathy are to be avoided.
```

Comment:
Rather than name-checking Obnoxious Aggression and Ruinous Empathy here, let's just focus on what Radical Candor means and why we want that: ```We strive for Radical Candor in our communications, where the intersection of caring personally and challenging directly allows us to foster a culture of productive guidance.```